### PR TITLE
Fix small typo

### PR DESCRIPTION
**Pull request by gmile**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -78,4 +78,4 @@
 
 #### Mix
 
-  * [Mix.Utils] `underscore/1` and `camelize/1` are deprecated
```
Comment (whatyouhide) [inline CHANGELOG.md]: Is this change killing the extra newline at the end of this file? Never seen a diff like this but just checked CHANGELOG.md and indeed there's an extra newline at the end of the file :\

Comment (josevalim) [inline CHANGELOG.md]: Yup, we should keep the new line at the end of the file. :)

Comment (whatyouhide) [inline CHANGELOG.md]: @josevalim not the final newline, there's an *extra* empty line before the final newline that we always keep. I think the extra line before the final newline should go, it's usually considered trailing whitespace?

**Comment by whatyouhide** [comment]
@gmile no worries, I was just having trouble understanding what that was happening in the diff but I checked out the PR locally and indeed it was removal of the extra newline before the final newline. All good, thanks a lot! 💟

**Pull request merged**